go/teams/transactions.go

```diff
@@ -642,6 +643,22 @@ func (tx *AddMemberTx) Post(ctx context.Context) (err error) {
 		if !found {
 			return fmt.Errorf("AddMemberTx.Post got a PerTeamKey but couldn't find a link with None to attach it")
 		}
+	} else { // If we didn't rotate the PTK, let's rebox the existing PTK for any of the new members
+		ekLib := g.GetEKLib()
+		if ekLib != nil {
```

Comment:
What's this? I don't like the idea of this silently not running if a test is set up improperly.

Reply:
this is unfortunate but for most tests we can't setup the EK stuff because of dependency cycles. are there any alternatives?